app/models/message_draft.rb

```diff
@@ -70,6 +70,7 @@ class MessageDraft < Message
 
   def update_content(parameters)
     metadata["data"] = parameters.to_h
+    self.html_visualization = format_html_visualization
```

Comment:
Toto mi pride podozrive. preco sa to nedeje v tom build_message_from_template?

Reply:
Pravda, nebolo to tuto spravne, presunula som.